### ISSUE TITLE
Add interface to query list of instruments ingested

### ISSUE THREAD

**Issue by garydoranjr** [JPLMLIA/pdsc#37]
Add the ability for a `PdsClient` to return a list of instruments that have been ingested by PDSC.

**Comment by Mirzyaaliii** [comment]
Yes, I agree. Having the list of instruments will make it easier for users.
